### PR TITLE
Users read

### PR DESCRIPTION
**Pull request by silverkd70**

### CODE REVIEW

**Comment by silverkd70** [comment]
- As a developer, I can setup my AnswersController and QuestionsController to include: ['user']
- As a developer, I can pull all the users from the database and save all the users into app.js as state
- As a developer, I can setup a render for question.user.alias in my QuestionCard.js
- As a developer, I can setup a render for question.user.alias in my QuestionShow.js
- As a developer, I can setup a render for answer.user.alias in my Answer.js
- As a user, I can see the alias name that belongs to the Question and Image post on the questionindex page.
- As a user, I can see the alias name that belongs to the Question and Image post on the card, and I can see the alias name that belongs to all the Answers comments on the snackbars.
- In order to have the correct user information that belongs to the corresponding questions and answers, the schema was modified and a rails db:reset was performed

**Comment by ghost** [comment]
Saw a merge conflict and resolved it! It was just an adjustment to a prop to include question.user.id.

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -88,26 +97,24 @@ class App extends React.Component {
...
-    .then((payload) => this.questionRead())
-    .catch((errors) => console.log("Question create errors:", errors));
-    
-};
-
+      .then((response) => {
+        if (response.status === 422) {
+          alert("You call that a question?! TRY AGAIN!");
+        }
+        console.log("look at me", response);
```
Comment (sjproctor) [inline app/javascript/components/App.js:112]: You can remove this console log. :)

**Review by sjproctor — APPROVED** [review]
Nice job! Just remove your console.log on the next branch.

**Pull request merged**